core/api/jrpc/jrpc_server.hpp

```diff
@@ -31,7 +31,9 @@ namespace kagome::api {
      * @param name rpc method name
      * @param method handler functor
      */
-    virtual void registerHandler(const std::string &name, Method method) = 0;
+    virtual void registerHandler(const std::string &name,
+                                 Method method,
+                                 bool unsafe = false) = 0;
```

Comment:
An enum would look better in method calls. 
registerHandler("name", ...method..., RpcSafety::Unsafe)
vs
registerHandler("name", ...method..., true)